### PR TITLE
Remove leftovers from removed disk concept.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

**Comment by geirst** [comment]
@vekterli FYI

File: storage/src/vespa/storage/bucketdb/bucketmanager.cpp [hunk 1]
```diff
@@ -243,16 +238,16 @@ BucketManager::updateMetrics(bool updateDocCount)
                 assert(bm != _metrics->bucket_spaces.end());
                 // No system with multiple bucket spaces has more than 1 "disk"
                 // TODO remove disk concept entirely as it's a VDS relic
-                bm->second->buckets_total.set(m.disk[0].buckets);
-                bm->second->docs.set(m.disk[0].docs);
-                bm->second->bytes.set(m.disk[0].bytes);
-                bm->second->active_buckets.set(m.disk[0].active);
-                bm->second->ready_buckets.set(m.disk[0].ready);
+                bm->second->buckets_total.set(m.count.buckets);
```
Comment (vekterli) [inline storage/src/vespa/storage/bucketdb/bucketmanager.cpp:241]: The above comments can probably be removed now

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**